src/core/hle/service/ac/ac.cpp

```diff
@@ -103,7 +103,7 @@ void Module::Interface::GetWifiStatus(Kernel::HLERequestContext& ctx) {
 
 void Module::Interface::GetInfraPriority(Kernel::HLERequestContext& ctx) {
     IPC::RequestParser rp(ctx, 0x27, 0, 2);
-    const std::vector<u8>& ac_config = rp.PopStaticBuffer();
+    [[maybe_unused]] const std::vector<u8>& ac_config = rp.PopStaticBuffer();
```

Comment:
I really like silencing the warnings. But maybe instead of [[maybe_unused]] it is just better to comment it out. E.g
```suggestion
    /*const std::vector<u8>& ac_config = */rp.PopStaticBuffer();
```

That way we don't forget to remove the [[maybe_unused]] when the function is properly implemented. But I'm not sure clang-format will let us do this nicely

Reply:
Compilers can warn about this. e.g. clang will issue a `-Wused-but-marked-unused` warning.

Reply:
then: never mind